test/connectSpec.js

```diff
@@ -183,6 +190,7 @@ describe('Connection methods', function () {
     })
 
     it('doesn\'t connect when the "account" is invalid', function (done) {
+      nock.removeInterceptor(this.accountNock.interceptors[0])
```

Comment:
Indeed, I also had problems with `.removeInterceptor(...)`. The example code from the `nock` docs does not work. After trying around a bit, I found `nock.removeInterceptor(this.accountNock.interceptors[0])` to be working.